src/default_panels/StyleImagesPanel.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ImageAccordion,
+  Radio,
+  TextEditor,
+  Numeric,
+  Section,
+  PositioningRef,
+  Dropdown,
+  MenuPanel,
+} from '../components';
+
+import {localize} from '../lib';
+
+const StyleImagesPanel = ({localize: _}) => (
+  <ImageAccordion canAdd>
+    <Radio
+      attr="visible"
+      options={[
+        {label: _('Show'), value: true},
+        {label: _('Hide'), value: false},
+      ]}
+    />
+    <TextEditor attr="source" label={_('Source')} show />
```

Comment:
yeah, we need the component hein..
would you mind creating the issue so we don't forget..

Reply:
yep. we can use `react-dropzone` I think, I used it for https://react-pivottable.js.org/ and it works great.

Reply:
https://github.com/plotly/react-plotly.js-editor/issues/333